itests/org.openhab.binding.modbus.tests/src/main/java/org/openhab/binding/modbus/tests/ModbusTcpThingHandlerTest.java

```diff
@@ -106,17 +91,11 @@ public void testTwoDifferentEndpointWithDifferentParameters() {
         thingConfig.put("timeBetweenTransactionsMillis", 1);
 
         final Bridge thing = createTcpThingBuilder("tcpendpoint").withConfiguration(thingConfig).build();
-        ThingHandlerCallback thingCallback = Mockito.mock(ThingHandlerCallback.class);
-        Mockito.doAnswer(invocation -> {
-            thing.setStatusInfo((ThingStatusInfo) invocation.getArgument(1));
-            return null;
-        }).when(thingCallback).statusUpdated(ArgumentMatchers.same(thing), ArgumentMatchers.any());
-
-        ModbusTcpThingHandler thingHandler = new ModbusTcpThingHandler(thing, () -> modbusManager);
-        thingHandler.setCallback(thingCallback);
-        thingHandler.initialize();
+        addThing(thing);
         assertThat(thing.getStatus(), is(equalTo(ThingStatus.ONLINE)));
 
+        ModbusTcpThingHandler thingHandler = (ModbusTcpThingHandler) thing.getHandler();
+        assert thingHandler != null;
```

Comment:
why are you using the `assert` keyword instead of `Assert.notNull(...)`?

Reply:
Thank you for that, would make sense to harmonize the assertions. Now hamcrest is used heavily but also plain assert and perhaps junit assert.

Coding conventions seem to prefer indeed junit Assert methods (assertNotNull) and hamcrest assertThat(new Object(), is(not(nullValue())));

I will switch to either hamcrest or junit assert*? I would prefer hamcrest since that has been used quite much already.

https://www.openhab.org/docs/developer/tests.html#assertions

Reply:
There is no reason you can't use both junit and hamcrest, in fact junit integrates hamcrest into it. As far as conventions I would suggest a simple Assert.assertNotNull(...) is better than the much more verbose hamcrest approach.

Reply:
Resolved in d214818.